src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/doc_viewer_transaction_overview/sub_components/transaction_duration_summary/index.tsx

```diff
@@ -8,78 +8,108 @@
  */
 
 import React from 'react';
-import {
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiHorizontalRule,
-  EuiSpacer,
-  EuiText,
-  EuiTitle,
-} from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiText, EuiTitle } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { Duration } from '@kbn/apm-ui-shared';
+import { Duration, DurationDistributionChart } from '@kbn/apm-ui-shared';
+import { ProcessorEvent } from '@kbn/apm-ui-shared';
 import { useRootTransactionContext } from '../../hooks/use_root_transaction';
+import { useTransactionLatencyChart } from '../../hooks/use_transaction_latency_chart';
+import { Section } from '../../../components/section';
+import { FieldWithoutActions } from '../../../components/field_without_actions';
 
 export interface TransactionDurationSummaryProps {
-  duration: number;
+  transactionDuration: number;
+  transactionName: string;
+  transactionType: string;
+  serviceName: string;
 }
 
-export function TransactionDurationSummary({ duration }: TransactionDurationSummaryProps) {
-  const { transaction, loading } = useRootTransactionContext();
+export function TransactionDurationSummary({
```

Comment:
`TransactionDurationSummary` and `SpanDurationSummary` are pretty much the same. I wonder if it would make sense to try to unify them in one single DurationSummary component.